spark/README.md

```diff
@@ -1,133 +0,0 @@
-# Apache Spark Helm Chart
```

Comment:
I understand we don't want to maintain config, but I would keep this README with the deleted configuration

Reply:
Do you mean the readme but without the configuration part?

Reply:
yes!